openprescribing/frontend/tests/fixtures/price_per_unit/formulation_swaps.csv

```diff
@@ -0,0 +1,12 @@
+The forma ,Code,Sept_Quantity,Formulation,Alternative formulation,Really equivalent?,,Alternative name,Alternative quantity,Number of words in formulation,Alternative code,DM+Dnonbioequivalent

```

Comment:
Is `The forma ` a mistake?

Reply:
Yes, quite possibly; but it's a mistake in the original Google Doc which I've downloaded verbatim so I'd rather not fiddle with it.